src/app/components/SaveJobButton/SaveJobButton.jsx

```diff
@@ -1,39 +1,43 @@
-import React from 'react';
+import React, { useState } from 'react';
 import axios from 'axios';
 import PrimaryButton from '../PrimaryButton';
+import SecondaryButton from '../SecondaryButton';
 
 const SaveJobButton = ({job}) => {
-  const getUser = () => {
-    const curPort = location.port;
-    axios({
-      method: 'GET',
-      withCredentials: true,
-      url: `http://localhost:${curPort}/user`,
-    }).then((res) => {
-      axios.post('/saveJob', {
-        data: {
-          job: job,
-          user: res.data
-        }
-      })
-    });
-  };
+  const [saveStarted, setSaveStarted] = useState(false);
 
   const handleSaveJobClick = (e) => {
+    setSaveStarted(true);
+  }
+
+  const handleInterestLevelClick = (e) => {
+    job.interest_level = e.target.textContent;
     axios({
       method: 'POST',
       url: '/saveJob',
       withCredentials:true,
       data: {
         job: job,
-        //user: res.data,
       },
     })
+    setSaveStarted(false);
   }
 
-  return(
-    <PrimaryButton text={'Save Job'} onClick={handleSaveJobClick} />
-  )
+  return (
+    <div>
```

Comment:
I'll reiterate Quinton's reminder to switch over to using <Box> instead of <div>